tasks/build.py

```diff
@@ -44,6 +44,7 @@
 _ERROR_GIT_APPLY = "git apply"
 _ERROR_GIT_CHECKOUT = "git checkout"
 _ERROR_GIT_CLONE = "curl"
```

Comment:
pre-existing, but shouldn't this be `git clone` instead of `curl`?

Reply:
Yeah, I guess so, I'll correct that after double checking